src/components/Contacts.tsx

```diff
@@ -2,21 +2,34 @@ import { useState, useEffect } from 'react';
 import api from '../services/api';
 
 import ContactCard from './ContactCard';
+import GridListToggle from './GridListToggle';
 
 import '../styles/Contacts.scss';
 
 const Contacts = () => {
     const [users, setUsers] = useState<any[] | null>(null);
+    const [viewOption, setViewOption] = useState('grid');
```

Comment:
no, it is just a simple string, anyway i will add type to the state. Thanks!

Reply:
If you use strings in conditions it is better to use enum for them.
In your case it might be 
```
enum VievType {
GRID = 'grid,
LIST = 'list'
}
```

and then you use it:
```
if(VievType.GRID) {
...
}
```

In that case you won't be able to have a typo. TypeScript won't allow you.